src/pages/trips/list/index.vue

```diff
@@ -34,18 +36,26 @@ if (store.origin && store.destination && store.date) {
   router.push({ path: '/search', replace: true });
 }
 
-getFilteredTrips({
-  date: store.date,
-  origin: store.origin,
-  destination: store.destination,
-})
-  .then((response) => {
-    trips.value = response.documents.filter(
-      (trip) => trip.totalPassengers - trip.passengerIds.length >= store.passengers
-    );
+const updateTrips = () => {
+  isLoading.value = true;
+  getFilteredTrips({
+    date: store.date,
+    origin: store.origin,
+    destination: store.destination,
+    conveniences: conveniences.value,
   })
-  .catch(captureApiException)
-  .finally(() => (isLoading.value = false));
+    .then((response) => {
+      trips.value = response.documents.filter(
+        (trip) => trip.totalPassengers - trip.passengerIds.length >= store.passengers
+      );
+    })
+    .catch(captureApiException)
+    .finally(() => (isLoading.value = false));
+};
+
+watch([origin, destination, date, passengers, conveniences], updateTrips, { deep: true });
```

Comment:
```suggestion
watch(tripSettingsStore, updateTrips, { deep: true });
```
проще то наверно уже так. Ноя бы вообще сделал по-другому, пусть компонент PageHeader делает `emit` при изменении фильтров, и как раз в него будет передаваться функция обновления